Test/unicodechars/comp/NativeNumbers.dfy.js.check

```diff
@@ -0,0 +1 @@
+// CHECK: Error: None of the types given in :nativeType arguments is supported by the current compilation target. Try supplying others.
```

Comment:
```suggestion
// CHECK-L: Error: None of the types given in :nativeType arguments is supported by the current compilation target. Try supplying others.
```

Reply:
That actually wouldn't work since the actual output is:

```
Test/comp/NativeNumbers.dfy(14,30): Error: None of the types given in :nativeType arguments is supported by the current compilation target. Try supplying others.
Test/comp/NativeNumbers.dfy(15,30): Error: None of the types given in :nativeType arguments is supported by the current compilation target. Try supplying others.
...
```

CHECK-L requires the whole trimmed line matches, not just part. We could add the source location prefixes, but that would just make this test more fragile, so I opted not to.

Reply:
Ah I did not know. In that, case you might just want to escape the dots since it's a regex :-)